### ISSUE TITLE
[Need inputs] Create a chapter in the Community Handbook for accessibilty considerations we commit to applying in all community events [online]

### ISSUE THREAD

**Issue by aleesteele** [the-turing-way/the-turing-way#3489]
### Summary

- During setting up a registration page for onboarding calls, we are having discussion in the #accessibility channel of our Slack workspace and have found a need to have a chapter in our handbook that describes the accessibility considerations we want to apply to all events rather than asking folks to tell us what accessibility needs they may have

### What needs to be done?

- [ ] Collect notes from the ongoing discussion and work from the onboarding call's registration form
- [ ] Collect accessibility considerations we want to offer 
  - CoC enforced for all calls
  - A co-host to support with technical glitches and additional question that may appear during the call
  - Link to slides and materials used during the call so that folks can follow them on their own - for example, visually impaired folks can follow the slides as they can't see the screen share
  - Live transcription via Zoom by default - and otter if we can
  - Shared document editable for all
  - Chat enabled for written input to the discussion
  - Other?
- [ ] Also collect accessibility considerations we can't offer and describe why, as well as how we can support participants
- [ ] clearly state what will we do with data and notes collected (GDPR)

### Who can help?

* Everyone
* Anne to help put this in the Community Handbook

---

**Comment by malvikasharan** [comment]
Thank you @LizHareDogs for already adding a lot to this discussion. Reposting her comment from Slack for the record:
> I have gotten some feedback from Erin Giglio, who has given me permission to give her credit by name. She agrees that the info should be available on the announcement (thanks, for the edit, Anne!). She used a better term than my "welcoming" for how the info on the announcement is received-- "access familiarity." The available info shows the org is familiar with the need for access and how to do it (hopefully). She doesn't mind checking off a box. She doesn't see a big difference between checking off a box and emailing, but these should only be for accessibility practices you don't do automatically.

**Comment by MarionBWeinzierl** [comment]
The RSE Society has an events EDI checklist - I cannot find it linked somewhere, but might be worthwhile asking them for it?

It is partly based (but adapted and extended) from the one the university of York has: https://www.york.ac.uk/media/abouttheuniversity/equality/documents/Inclusive-and-accessible-events-good-practice-guidelines-updated-May-2022.pdf

**Comment by MarionBWeinzierl** [comment]
There is one thing I find missing in checklists and guidance I have seen so far, and that is considerations around mental health. Of course it is impossible to be proactive around all possible issues. But, just as one example, it could help if there was a designated (trained) person at a designated place who people can reach out to, for example if they feel an anxiety or panic attack coming up. If they are (quite literally) feeling they are going to die or something else terrible will happen, if they need "to get out of here" as claustrophobia is sinking down on them, not having to panicly look for a way out or searching for someone who can help can in itself have a calming effect. 

I think this is a longer discussion with lots of different aspects to consider, and I am not sure whether there is an easy (quite certainly not)  or good solution to be found. But I would think it could make a big difference to a lot of people to have this discussion and to do the best we can.

**Comment by malvikasharan** [comment]
That's a very good point.

By default our events are online so I didn't even consider the online vs
other events for this discussion. Should make that distinction clearly for
sure in our guidance.

How can we translate mental health considerations for online setting?
- adding breaks in longer meetings
- making it clear that folks can contact a designated person
- if they are not feeling well, they can leave anytime
- other considerations?

**Comment by KirstieJane** [comment]
@MarionBWeinzierl - this is a great point. When we've run events with our autistic collaborators for a citizen science project we have made sure to have a designated quiet room booked in case anyone needs to step out for a cool down.

This is often difficult to manage at the Turing Institute because the rooms are so in demand but there is a wellness room that we could maybe point people to? We just wouldn't be able to _guarantee_ that no one else would be using it but it is often free....

**Comment by KirstieJane** [comment]
I wonder if there's an online version of this where we have a breakout room open that anyone can step into if they need to?

To be honest that's probably pretty complicated (I feel like folks would be confused about whether they're supposed to join the breakout room or not!) so maybe there's a section for online facilitation where we emphasise that anyone can turn off their video and step away from the computer whenever they need to? They don't need to wait for a break.

**Comment by LizHareDogs** [comment]
I like Kirstie's suggestion of telling people they can always turn off video and wakl away. Especially as events are being recorded-- people will not miss learning stuff if they have to step away to decompress.

**Comment by aleesteele** [comment]
Hi @MarionBWeinzierl, @malvikasharan, @KirstieJane, @geekygirldawn @LizHareDogs! 

Thank you all so much for your notes and responses, apologies I'm a few days late. I was recovering from a flu last week and a bit slow all around. 😷 These are really valuable to keep in mind in addition to other work that the Accessibility Working Group (@the-turing-way/accessibility) is working on in the project - like #31 #3426 #3307 - chapters and resources related to accessibility and events (in case you're not aware Dawn and Marion!). 

Marion, thank you for this important question around mental health and having spaces/people/resources that might help! I  really like Kirstie's idea that verbal flag that anyone can step away from their devices or turn off their videos if they'd like. During the previous book dash - when people felt they needed breaks from screens, they tended to leave the zoom entirely instead of wanting to join a quiet coworking room. We also tend to normalise a video-on culture in the project (accidentally or not), so having more explicit sign-posting of being able to step away when you'd like to is important. In terms of using the wellness room and flagging it for attendees to use at the Turing, that's definitely something for @AlexandraAAJ and myself to investigate for next book dash. Thanks for the suggestion 😄 

Dawn, thanks for the links to the badges too - this is something that would definitely be valuable for our book dash organising committee! Thanks! 

This week I'm going to add some notes here from our discussion on the accessibility slack channel for posterity, and to help with the chapter-writing process. I got started on a draft last week - while what we currently offer may be different from what we'd **like** to offer in the future, it's a first step!

**Comment by LizHareDogs** [comment]
Reading Ann's mention that the culture tends to be "video on" reminds me that this is also a bandwidth issue. When participants join Zoom from locations with less bandwidth, sometimes their ability to participate is enhanced when there is less video data. I tend to turn mine off is I suspect that is happening.

**Comment by malvikasharan** [comment]
I agree. I think in most public events most places I have seen, a general behaviour is video off (to save data or otherwise to avoid the awkwardness of not knowing folks and not having a real opportunity to speak up). When recording, we also recommend video off to accidentally avoid being on recording if folks prefer.
I think Anne might be referring to smaller Working Group meetings, Collan Cafe and Book Dash events (with breakouts) where people tend to know each other and actively want to talk. Though bandwidth issues stand true there as well.

**Comment by aleesteele** [comment]
Hi folks, closing this as it was addressed and merged (in a basic form) in #3491 - thank you all so much!
